rpc.go

```diff
@@ -25,17 +25,36 @@ type Rpc struct {
 
 type opts struct {
 	cors      string
+	headers   string
```

Comment:
`headers` and `methods` are not clear, nobody will understand their purpose. I suggest to create a new structure `cors` with fields `allowOrigin`, `allowHeaders`, `allowMethods` and use it in `opts`.
for example
```go
type opts struct {
    cors *cors
    maxMemory int64
}

type cors struct {
 // ...
}

```